# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp
# Commit: 91d8f0f829fd
# Full Hash: 91d8f0f829fd2ae6ec234a6a3f372bf503ed2fbd
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r 91cb8ee8ddd8 -r 91d8f0f829fd gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp
--- a/gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp	Thu Nov 05 19:26:21 2020 +0000
+++ b/gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp	Thu Nov 05 20:18:13 2020 +0000
@@ -44,6 +44,7 @@
       mCompositor(aCompositor),
       mContext(aContext) {
   MOZ_ASSERT(mContext);
+  mSyncObject = mCompositor->GetSyncObject();
 }
 
 RenderCompositorD3D11SWGL::~RenderCompositorD3D11SWGL() {